WireAPI/Sources/WireAPI/Models/Conversation/Conversation.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public struct Conversation {
```

Comment:
It'd be nice to add some doc comments to this public type. Also I think we could also have  more stricter types to replace the strings. What do you think?

Reply:
I agree, added some comments on the structs: d7df938c01aa26a0fbe2ca8b3093591a88cc1c55
Unfortunately I don't have knowledge to document all properties.